src/index.js

```diff
@@ -40,6 +40,12 @@ app.on('will-finish-launching', () => {
 })
 
 function handleError (err) {
+  // Ignore network errors that might happen during the
+  // execution.
+  if (err.stack.includes('net::')) {
+    return
+  }
+
   logger.error(err)
```

Comment:
could we move this logger before the `if`, so its always logged?

Reply:
I was just thinking about ignoring the noise caused by Internet issues...

Reply:
Just to be sure I got this right, this is when ipfs-desktop tries to access API that is offline, or are there other uses of this code path?

Reply:
AFAIK, it's when we try to access the API and it is offline. However, it shouldn't happen many times.

On previous occasions, this would happen with Auto Updater, when we didn't catch the errors.